packages/client/ui/Tooltip/TooltipContent.tsx

```diff
@@ -0,0 +1,24 @@
+import {Content, Portal} from '@radix-ui/react-tooltip'
+import * as React from 'react'
+import {twMerge} from 'tailwind-merge'
+import {forwardRadix} from '../fordwardRadix'
+
+export const TooltipContent = forwardRadix<typeof Content>(
```

Comment:
added forwardRadix to get strong types without having to dig into the radix components to figure out props & element types